Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -57,36 +59,70 @@ static void* mmap_with_name(void* addr, size_t length, int prot, int flags, int
 
 namespace ELF {
 
-static bool s_always_bind_now = false;
+static bool s_always_bind_now = true;
```

Comment:
Issues with PLT relocations? Those are annoyingly tricky...

Reply:
They sure are, I set this flag so we can not worry about these right now and make things easier to debug.
I'll switch back to doing lazy binding once things stabilize a bit, and it should only be a matter of flipping this flag and passing `RTLD_LAZY` to `ELF::DynamicLoader` I think, because you've already implemented everything needed there :smiley: